<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fifteenPuzzleConfig</class>
 <widget class="QWidget" name="fifteenPuzzleConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Fifteen Puzzle</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Size:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>size</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="size">
     <property name="suffix">
      <string> pieces wide</string>
     </property>
     <property name="minimum">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QRadioButton" name="rb_identical">
     <property name="text">
      <string>Use plain pieces:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="KColorCombo" name="color"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QRadioButton" name="rb_split">
     <property name="text">
      <string>Use custom image:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="KUrlRequester" name="urlRequester">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="fileDialogModality">
      <enum>Qt::NonModal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lb_showNumbers">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Show numerals:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>cb_showNumerals</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="cb_showNumerals">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rb_split</sender>
   <signal>toggled(bool)</signal>
   <receiver>urlRequester</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_split</sender>
   <signal>toggled(bool)</signal>
   <receiver>color</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_split</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_showNumerals</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_split</sender>
   <signal>toggled(bool)</signal>
   <receiver>lb_showNumbers</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
